script.js

```diff
@@ -274,3 +274,42 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
   }
 });
+
+// Local Storage Practice
+const storageValues = {
+  user: '',
+  email: '',
+  message: '',
+};
+
+document.getElementById('user').addEventListener('keyup', () => {
+  storageValues.user = document.getElementById('user').value;
+  storageValues.email = document.getElementById('email').value;
+  storageValues.message = document.getElementById('message').value;
+  localStorage.setItem('contactDetails', JSON.stringify(storageValues));
+});
+
+document.getElementById('email').addEventListener('keyup', () => {
+  storageValues.user = document.getElementById('user').value;
+  storageValues.email = document.getElementById('email').value;
+  storageValues.message = document.getElementById('message').value;
+  localStorage.setItem('contactDetails', JSON.stringify(storageValues));
+});
+
+document.getElementById('message').addEventListener('keyup', () => {
+  storageValues.user = document.getElementById('user').value;
+  storageValues.email = document.getElementById('email').value;
+  storageValues.message = document.getElementById('message').value;
+  localStorage.setItem('contactDetails', JSON.stringify(storageValues));
+});
```

Comment:
- [ ] Excellent work, team; the project is working as expected 👏. However, I believe this code violates the DRY (Don't Repeat Yourself) principle, and I believe it would be tedious to repeat the same code over and over when have more value, right? ☺️ So I would kindly suggest that you fix this, and one way to do so is as follows. 
```
const updateStorage = () => {
  storageValues.user = document.getElementById('user').value;
  storageValues.email = document.getElementById('email').value;
  storageValues.message = document.getElementById('message').value;
  localStorage.setItem('contactDetails', JSON.stringify(storageValues));
};

document.getElementById('user').addEventListener('keyup', updateStorage);
document.getElementById('email').addEventListener('keyup', updateStorage);
document.getElementById('message').addEventListener('keyup', updateStorage);
```